examples/contract_template/README.md

```diff
@@ -0,0 +1,87 @@
+# Athena smart contract template
+
+This template shows the bare minimum that a smart contract must currently implement
+to be deployed and ran on the Athena-enabled test network.
+
+The contract implements 2 necessary methods:
+ - spawn
+ - verify
+
+Please check the comments in the [code](src/contract.rs) for explanations
+
+## Prerequisites
+### Rust installed
+The template is implememented in the Rust programming language and a working rust toolchain is needed. Follow the steps on https://rustup.rs too install it.
+
+### Athena toolchain
+It's needed to build the contract code to the Risc-V RV32EM target. 
+This target is not yet natively supproted by the Rust toolchain. 
```

Comment:
We have a specific target. It is not supported by the Rust toolchaing, but upstream rust toolchain supports RISC-V RV32EM. It does not support our platform.

My suggestion would be:
```
It's needed to buld the contract code to the Athena target (which is RISC-V RV32EM architecture targeted to Athena platform).
We maintain a specific toolchain for this target at ...
```